src/libmongoc/tests/unified/entity-map.c

```diff
@@ -365,6 +392,9 @@ entity_client_new (entity_map_t *em, bson_t *bson, bson_error_t *error)
             mongoc_apm_set_command_failed_cb (callbacks, command_failed);
          } else if (0 == strcmp (event_type, "commandSucceededEvent")) {
             mongoc_apm_set_command_succeeded_cb (callbacks, command_succeeded);
+         } else if (is_unsupported_event_type (event_type)) {
+            MONGOC_DEBUG ("Skipping observing unsupported event type: %s", event_type);
```

Comment:
Would it make sense to error here and require the test be explicitly skipped?

Reply:
I'd argue it still has some value to run the test even if we cannot make assertions on CMAP events.